mercadopago.gemspec

```diff
@@ -20,12 +20,12 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   # specify any dependencies here:
-  s.add_dependency 'json', '>= 1.4.6'
-  s.add_dependency 'faraday', '>= 0.9.0'
-  s.add_development_dependency 'pry'
-  s.add_development_dependency 'rake'
-  s.add_development_dependency 'byebug'
-  s.add_development_dependency 'test-unit'
-  s.add_development_dependency 'vcr'
-  s.add_development_dependency 'webmock'
+  s.add_dependency 'json', '~> 1.4', '>= 1.4.6'
```

Comment:
We can remove the second part: `>= 1.4.6`. `~> 1.4` should already attempt to use the latest patch available.